Packages/install_toggle

```diff
@@ -10,6 +10,53 @@ cd Packages
 
 echo "** install toggle **"
 set -x
+
+# first get the dependencies
+apt install -y \
+ gir1.2-clutter-1.0=1.26.0+dfsg-10.1 \
+ gir1.2-cogl-1.0=1.22.2-2.1 \
+ gir1.2-coglpango-1.0=1.22.2-2.1 \
+ gir1.2-mash-0.3-0=0.3.0-1 \
+ gir1.2-mx-2.0=1.99.4-1.1 \
+ libclutter-1.0-0=1.26.0+dfsg-10.1 \
+ libclutter-1.0-common=1.26.0+dfsg-10.1  \
+ libclutter-imcontext-0.1-0=0.1.4-3.1 \
+ libclutter-imcontext-0.1-bin=0.1.4-3.1 \
+ libcluttergesture-0.0.2-0=0.0.2.1-7.2 \
+ libcogl-common=1.22.2-2.1 \
+ libcogl-pango20=1.22.2-2.1 \
+ libcogl-path20=1.22.2-2.1 \
+ libcogl20=1.22.2-2.1 \
+ libmash-0.3-0=0.3.0-1 \
+ libmx-2.0-0=1.99.4-1.1 \
+ libmx-bin=1.99.4-1.1 \
+ libmx-common=1.99.4-1.1 \
+ python-networkmanager \
+ python-gi-cairo \
+ python-tornado \
+
+apt-mark hold \
+ gir1.2-clutter-1.0 \
+ gir1.2-cogl-1.0 \
+ gir1.2-coglpango-1.0 \
+ gir1.2-mash-0.3-0 \
+ gir1.2-mx-2.0 \
+ libclutter-1.0-0 \
+ libclutter-1.0-common \
+ libclutter-imcontext-0.1-0 \
+ libclutter-imcontext-0.1-bin \
+ libcluttergesture-0.0.2-0 \
+ libcogl-common \
+ libcogl-pango20 \
+ libcogl-path20 \
+ libcogl20 \
+ libmash-0.3-0 \
+ libmx-2.0-0 \
+ libmx-bin \
+ libmx-common \
+
+pip install requests pyconnman
```

Comment:
This line belongs in toggle's setup rather than here

Reply:
We should not use pyconnman anywhere - it will fail no matter who uses it, as we're using network-manager. As NetworkManager and connman are mutually exclusive, this means that we should rather remove this dependency from the underlying package.

Reply:
Perhaps we should not be using it. However, on my BBB, running the image I generated yesterday, I find that it is connman and not Network Manager that is running.

Some selected entries:
```
  connman.service                                                                                 loaded active running   Connection service       
  ifup@eth0.service                                                                               loaded active exited    ifup for eth0            
● ifup@usb0.service                                                                               loaded failed failed    ifup for usb0            
● mjpg.service                                                                                    loaded failed failed    Mjpg streamer            
● motd-news.service                                                                               loaded failed failed    Message of the Day       
  netfilter-persistent.service                                                                    loaded active exited    netfilter persistent confi
  networkd-dispatcher.service                                                                     loaded active running   Dispatcher daemon for syst
● networking.service                                                                              loaded failed failed    Raise network interfaces 
● NetworkManager-wait-online.service                                                              loaded failed failed    Network Manager Wait Onlin
  NetworkManager.service                                                                          loaded active running   Network Manager          
  nginx.service                                                                                   loaded active running   A high performance web ser
  nmbd.service                                                                                    loaded active running   Samba NMB Daemon         
  octoprint.service                                                                               loaded active running   OctoPrint                
  redeem.service                                                                                  loaded active running   The Replicape Daemon     
```
Note that I do not have a camera in this setup. So the failure of the `mjpg.service` is expected.

Reply:
@goeland86 objects to any use of `connman`. And I think the issue should be pushed over to the Toggle repository which should be responsible for doing the `pip install`s as a part of `setup.py`

Reply:
Yes - I would say remove the dependency from `Umikaze`'s setup, while we work on fixing Toggle in the meantime.

Reply:
Still unresolved ... is handled by setup in the Toggle repo.

Reply:
Toggle 1.3.x does not.

Reply:
No, `develop` is the current "working" branch. But that will revert to a newer `1.3.x` when we complete a release. We are setting up the build system to handle current and future builds, not past ones.

And, before you say that we have to be able to use the "stable" branch of one component in an "experimental" setting for another, let me point out that that fails when there is a change in the API between the components. So if, for example, OP improves the interface (as I have heard that `OP 1.4` will do), we will have to change Redeem at the same time so that it will use the new interface.

Reply:
@Wackerbarth OP `1.4` will modularize the interface - giving us the opportunity to write our own custom module instead of the serial to socat interface currently in use - but it will still be backwards compatible.

Reply:
So... what is the actionable feedback here? Do we want Umikaze to build with Toggle develop right now? Do I leave this line here so we can keep building working images with `1.3.x` ?